projects/ngx-scroll-to/package.json

```diff
@@ -55,11 +55,9 @@
     "node": ">=8.0.0",
     "npm": ">=6.0.0"
   },
-  "dependencies": {
-    "tslib": "^2.0.0"
-  },
   "peerDependencies": {
     "@angular/common": "^8.0.0 || ^9.0.0 || ^10.0.0",
-    "@angular/core": "^8.0.0 || ^9.0.0 || ^10.0.0"
+    "@angular/core": "^8.0.0 || ^9.0.0 || ^10.0.0",
```

Comment:
what about v11?

Reply:
I haven't tested with v11 or attempt to upgrade to it. Feel free to do it :)

Reply:
v11 works for us, v12 is out by the way, haven't testd..

Reply:
v12 also works. I suggest to simply add >= 8.0.0 and add a note about it in the readme to say that it is partially maintained and use at your own risk...